mathematical/stats.py

```diff
@@ -85,10 +86,10 @@ def std_none(dataset, ddof=1):
 	dataset = utils.remove_zero(dataset)
 	print(dataset)
 
-	return numpy.nanstd(dataset, ddof=ddof)
+	return float(numpy.nanstd(dataset, ddof=ddof))
 
 
-def median_none(dataset):
+def median_none(dataset: Sequence[Union[float, bool, None]]):
```

Comment:
Please can you add a return type of `float`, and cast the return value on line 106 to float?